tests/.terraform.lock.hcl

```diff
@@ -0,0 +1,60 @@
+# This file is maintained automatically by "terraform init".
```

Comment:
we should not be including this either.
This is a test, so it should track whatever is current given that reflects whatever anyone will use at any given moment.

Reply:
I dont feel strongly on checking this file but, we are not tracking what is current. all this file does is provide the available `sha` for the specified versions.

Reply:
we don't need to... we tend to run a deploy against this often enough that any break would be caught.
If we really care, we can add a weekly CI check to guarantee it always works but that would mean we actually get notified on failure and do something about it.
Tracking means little if we aren't pinning the versions.

Reply:
the checksum always occurs since its using the registry to download and verify.
the lock makes us use the version we did the initial `init` with. Unless we do an `init -upgrade` we will never be testing against them.